### PR TITLE
Feature/images

### PR DESCRIPTION
**Pull request by danielcfuentes**
## Description

This pull request works on bieng able to save an image that a user uploads to the database and then when the post is created and displayed the images are displayed and fetched from the database.

what will be done in later PRs and not included here:
The next pull request is me working on using React UI components.

## Milestones
MIlestones 2 and 3 for posting and user feed

## Resources
<links to tutorials, code snippets, inspirations>
- https://devcamp.com/trails/comprehensive-react-development-tutorial/campsites/277/guides/working-form-data-react
- https://expressjs.com/en/resources/middleware/multer.html
- https://www.cloudflare.com/learning/cloud/what-is-blob-storage/

## Test Plan
demo video:
https://www.loom.com/share/2154a4231eca4252a700a3e9418a350b?sid=91fdce58-f471-4b15-8a1f-86bbfae85ae9

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -24,55 +25,99 @@ function authenticateToken(req, res, next) {
...
-        user: { connect: { username: req.user.name } },
-        images: {
-          create: imageUrls.map((url) => ({ url })),
+app.post(
+  "/posts",
+  authenticateToken,
+  upload.array("images"),
+  async (req, res) => {
+    const { title, content } = req.body;
+    const images = req.files || []; // Provide a default empty array if req.files is undefined
```
Comment (janicetsai1) [inline backend/server.js]: you can move this down into the try statement since if title is not provided, then it would just return 400 error

**Review by janicetsai1 — APPROVED** [review]

**Comment by janicetsai1** [comment]
overall looks good! left one comment

**Review by jayeshdoolani — APPROVED** [review]

File: backend/server.js (hunk 1 above)
Comment (danielcfuentes) [reply backend/server.js]: Fixed! than you

**Comment by gwh59** [comment]
lgtm as well!

**Pull request merged**